src/parameters/update_parameters.jl

```diff
@@ -403,7 +408,7 @@ function update_container_parameter_values!(
     # if the keys have strings in the meta fields
     parameter_array = get_parameter_array(optimization_container, key)
     parameter_attributes = get_parameter_attributes(optimization_container, key)
-    _update_parameter_values!(parameter_array, parameter_attributes, T, model, input)
+    _update_parameter_values!(parameter_array, parameter_attributes, U, model, input)
```

Comment:
Why is this Update Parameter for FixValueParameter is not being tested?

Reply:
Tests added in [192dc3c](https://github.com/NREL-Sienna/PowerSimulations.jl/pull/1130/commits/192dc3c02a1fbc41631abf9d6f41fca946efd826)